README.md

```diff
@@ -19,7 +19,6 @@ There are multiple demos in this repo that showcase Kafka stream processing on t
 | [microservices-orders](microservices-orders/README.md) |   [Y](microservices-orders/README.md)   |   N    | integrates the [Microservices Orders Demo Application](https://github.com/confluentinc/kafka-streams-examples/tree/5.0.x/src/main/java/io/confluent/examples/streams/microservices) into Confluent Platform
 | [music](music/README.md)                   |   [Y](music/README.md)   |   [Y](music/README.md)    | KSQL version of the [Kafka Streams Demo Application](https://docs.confluent.io/current/streams/kafka-streams-examples/docs/index.html)
 | [mysql-debezium](mysql-debezium/README.md) |   [Y](mysql-debezium/README.md)   |   N    | end-to-end streaming ETL with KSQL for stream processing using the [Debezium Connector for MySQL](http://debezium.io/docs/connectors/mysql/)
-| [oracle-ksql-elasticsearch](oracle-ksql-elasticsearch/README.md) |   N   |   [Y](oracle-ksql-elasticsearch/README.md)    | enrich event stream data from Oracle, enriching it with KSQL, and then stream into Elasticsearch
```

Comment:
@rmoff : what do you think about leaving this in the README table and pointing it to `demo-scene` so users can still find it?